.github/dependabot.yml

```diff
@@ -3,14 +3,29 @@ updates:
 - package-ecosystem: "gradle"
   directory: "/"
   schedule:
-    interval: "daily"
+    interval: "weekly"
+	groups:
```

Comment:
Is this too much indentation? Or did tabs sneak in here?

Reply:
Sigh, thanks. Better now?

(I figured out a while back that I end up with tabs when I `set paste` in Vim....)